README.md

```diff
@@ -84,6 +85,15 @@ artifacts:
       - "./artifacts"
 ```
 
+### Run MATLAB Build
+Use the [MATLAB build tool](https://www.mathworks.com/help/matlab/matlab_prog/overview-of-matlab-build-tool.html) to run the build plan returned by a file named `buildfile.m` in the root of your repository. Run the default tasks in your plan as well as all the tasks on which they depend.
```

Comment:
This doesn't quite match the comment that we used on the template submission, but I like both. I just want to make sure that it's not an issue that they aren't identical.

Reply:
I think it's better if the comment align. Updated the text.